app/components/edit-user/template.hbs

```diff
@@ -15,6 +15,13 @@
         value={{this.userChangeset.username}}
         {{on 'input' (fn this.onPropertyChange 'username')}}
       />
+      <div class='validation-container'>
+        {{#if this.userChangeset.error.username}}
+          <span class='validation-err'>
+            {{this.userChangeset.error.username.validation}}
+          </span>
+        {{/if}}
+      </div>
```

Comment:
How about create separate component for that? Something like:
```hbs
<ValidationContainer @error={{this.userChangeset.error.username}} />
```
`/validation-container/template.hbs`:
```hbs
<div class='validation-container'>
  {{#if @error}}
    <span class='validation-err'>
      {{@error.validation}}
    </span>
  {{/if}}
</div>
```